### ISSUE TITLE
Add a warning explaining the need for a default route for Kubernetes clusters

### ISSUE THREAD

**Issue by atanasdinov** [suse-edge/suse-edge.github.io#308]

**Comment by ranjinimn** [comment]
@atanasdinov Can you please provide more information for this issue?

**Comment by atanasdinov** [comment]
Yeah, sorry for not putting a description. We basically want to ensure that our docs also list the k3s and RKE2 route prerequisites:
- https://docs.rke2.io/install/airgap#prerequisites
- https://docs.k3s.io/installation/airgap#default-network-route

**Comment by ranjinimn** [comment]
@atanasdinov Thank you for this information. Can you please let me know the sections where you would like these prerequisites to be included? Do you want me to include these as links in the relevant sections?
